llvm/docs/LangRef.rst

```diff
@@ -16625,20 +16639,34 @@ type.
 
 Semantics:
 """"""""""
-Follows the IEEE-754 semantics for maxNum except for the handling of
-signaling NaNs. This matches the behavior of libm's fmax.
+Follows the IEEE-754 semantics for minNum, except that -0.0 < +0.0 for the purposes
```

Comment:
Same as above, this changed between IEEE-754 2008 and 2019

Reply:
I try to not use the words like `change` here.  In the IEEE754 wordings, 2019 removes `minNum/maxNum`, and introduces new `minimumNumber/maximumNumber`.

Reply:
The intrinsic now has the semantics as IEEE-754 2019 minimumNumber. The 2008 mention of minNum can be the footnote.